build.gradle

```diff
@@ -88,7 +88,6 @@ dependencies {
     modRuntimeOnly "com.terraformersmc:modmenu:${project.mod_menu_version}"
 
     // CaffeineMC optimization mods
-    modRuntime "com.github.CaffeineMC:sodium-fabric:${project.sodium_version}"
```

Comment:
Why was sodium removed from debug runtime?

Reply:
I don’t know why, but the build is failing because of it, it seems to be not used anywhere, I decided to delete it